Update WW1 Legal docs (#8192)
zkSNACKs/WalletWasabi: WalletWasabi/Legal/Assets/LegalDocuments.txt

    If You find any reason to violate the law during Your transaction (for example, in a transaction with a third party), please let us know at one of the contacts listed at the end of this document.
    The Service Provider shall assist the investigation in any case, if so instructed by an authorized body, a final court judgment or a final regulatory decision.

> Coin Filtering 

Reviewer: ```suggestion
    COIN FILTERING
```

> ZkSNACKs Ltd. may execute illicit activity checking and control via a contracted third party solely in its coinjoin coordination services. ZkSNACKs Ltd. may suspend your UTXOs’ access to the coinjoin services, with immediate effect for any reason - including but not limited to Illicit or prohibited activities, applicable sanctions programs, or any crime or money-laundering activity - at its sole discretion and is under no obligation to disclose the details of its decision to take such action with you. In this case you are not permitted to use the relevant/high-risk bitcoin UTXO to reach the coinjoin services. 

Reviewer: ```suggestion
    zkSNACKs Ltd. may execute illicit activity checking and control via a contracted third party solely in its coinjoin coordination services. zkSNACKs Ltd. may suspend your UTXOs’ access to the coinjoin services, with immediate effect for any reason - including but not limited to illicit or prohibited activities, applicable sanctions programs, or any crime or money-laundering activity - at its sole discretion and is under no obligation to disclose the details of its decision to take such action with you. In this case you are not permitted to use the relevant/high-risk bitcoin UTXO to reach the coinjoin services.
```

Reviewer 2: Isn't it capitalized at beginning of sentence?

> Wasabi may execute illicit activity checking and control via a contracted third party solely in its CoinJoin Coordination Services. Wasabi may suspend your UTXOs’ access to the CoinJoin Services, with immediate effect for any reason - including but not limited to Illicit or Prohibited Activities, applicable sanctions programs, or any crime or money-laundering activity - at its sole discretion and is under no obligation to disclose the details of its decision to take such action with you. In this case you are not permitted to use the relevant/high-risk Bitcoin UTXO to reach the CoinJoin Services. 
> You acknowledge that Wasabi's decision to take certain actions, including suspending for any reason at our sole discretion, may be based on confidential criteria that are essential to Wasabi's risk management and security protocols. You agree that Wasabi is under no obligation to disclose the details of its risk management and security procedures to you.
> Your access with the relevant Bitcoin UTXOs to the CoinJoin Services will be permanently suspended.

Reviewer 2: ```suggestion
ZkSNACKs may execute illicit activity checking and control via a contracted third party solely in its coinjoin coordination services. ZkSNACKs may suspend your UTXOs’ access to the coinjoin services, with immediate effect for any reason - including but not limited to Illicit or prohibited activities, applicable sanctions programs, or any crime or money-laundering activity - at its sole discretion and is under no obligation to disclose the details of its decision to take such action with you. In this case you are not permitted to use the relevant/high-risk bitcoin UTXO to reach the coinjoin services. 
You acknowledge that zkSNACKs' decision to take certain actions, including suspending for any reason at our sole discretion, may be based on confidential criteria that are essential to zkSNACKs' risk management and security protocols. You agree that zkSNACKs is under no obligation to disclose the details of its risk management and security procedures to you.
Your access with the relevant bitcoin UTXOs to the coinjoin services will be permanently suspended.
```

> You acknowledge that zkSNACKs Ltd.'s decision to take certain actions, including suspending for any reason at our sole discretion, may be based on confidential criteria that are essential to zkSNACKs Ltd.'s risk management and security protocols. You agree that zkSNACKs Ltd. is under no obligation to disclose the details of its risk management and security procedures to you.
> Your access with the relevant bitcoin UTXOs to the coinjoin services will be permanently suspended.

Reviewer: ```suggestion
    You acknowledge that zkSNACKs Ltd.'s decision to take certain actions, including suspending for any reason at our sole discretion, may be based on confidential criteria that are essential to zkSNACKs Ltd.'s risk management and security protocols. You agree that zkSNACKs Ltd. is under no obligation to disclose the details of its risk management and security procedures to you.
    Your access with the relevant bitcoin UTXOs to the coinjoin services will be permanently suspended.
```

    You and the Service Provider agree to arbitrate any dispute arising under or in connection with these Terms or Your use of our Services, except for disputes in which either party seeks equitable and other relief for any alleged infringement or unlawful use of copyrights, trademarks, trade names, logos, trade secrets, patents or other intellectual property rights. Arbitration prevents You from suing in court or from having a jury trial. You and the Service Provider agree to notify each other in writing of any dispute within thirty (30) days of when it arises. Notice to the Service Provider shall be sent to legal@zksnacks.com.

    You and the Service Provider further agree:

> - to attempt informal resolution prior to any demand for arbitration;

Reviewer: ```suggestion
    - to attempt informal resolution prior to any demand for arbitration;
```

>     - to attempt informal resolution prior to any demand for arbitration;

Reviewer: ```suggestion
     - to attempt informal resolution prior to any demand for arbitration;
```

     - that the seat or legal place of any arbitration will be Republic of Seychelles;
     - that arbitration will be conducted confidentially by a single arbitrator appointed by law of Republic of Seychelles;
     - the arbitration shall be conducted in accordance with the rules of International Chamber of Commerce; and
     - that the arbitration in Republic of Seychelles will have exclusive jurisdiction over any appeals of an arbitration award.

>    Other than class procedures and remedies discussed below, the arbitrator has the authority to grant any remedy that would otherwise be available in court.

Reviewer: ```suggestion
    Other than class procedures and remedies discussed below, the arbitrator has the authority to grant any remedy that would otherwise be available in court.
```

Reviewer 2: Why?

    Whether the dispute is heard in arbitration, You and the Service Provider will not commence against the other any class action, class arbitration or representative action or proceeding.

9.3 GOVERNING LAW AND JURISDICTION

>   These Terms and any dispute or claim between You and the Service Provider arising out of or in connection with these Terms or any terms incorporated into these Terms by reference or their subject matter or formation (including non-contractual disputes or claims) will be governed by and construed in accordance with the laws of Republic of Seychelles, without giving effect to any conflict of laws principles that may provide for the application of the law of another jurisdiction. Subject to the “Arbitration” section above, the arbitration of Republic of Seychelles shall have exclusive jurisdiction to settle any dispute or claim between You and the Service Provider arising out of or in connection with these Terms or any terms incorporated into these Terms by reference or their subject matter or formation (including non-contractual disputes or claims).

Reviewer: ```suggestion
    These Terms and any dispute or claim between You and the Service Provider arising out of or in connection with these Terms or any terms incorporated into these Terms by reference or their subject matter or formation (including non-contractual disputes or claims) will be governed by and construed in accordance with the laws of Republic of Seychelles, without giving effect to any conflict of laws principles that may provide for the application of the law of another jurisdiction. Subject to the “Arbitration” section above, the arbitration of Republic of Seychelles shall have exclusive jurisdiction to settle any dispute or claim between You and the Service Provider arising out of or in connection with these Terms or any terms incorporated into these Terms by reference or their subject matter or formation (including non-contractual disputes or claims).
```

COUNTRY OF RESIDENCE
====================

> In case You are a Gibraltar or Republic of Seychelles resident and willing to use Wasabi Wallet please inform us in advance on the following e-mail address: legal@zksnacks.com.

Reviewer: ```suggestion
    In case You are a Gibraltar or Republic of Seychelles resident and willing to use Wasabi Wallet please inform us in advance on the following e-mail address: legal@zksnacks.com.
```

Why here we mention both Gibraltar and Republic of Seychelles?

    zkSNACKs Ltd.
    www.zksnacks.com(*)

> (*) We recommend You to prefer Tor Browser over other browsers in order to protect Your privacy while visiting our websites.

Reviewer 2: ```suggestion
(*) We recommend you to prefer Tor Browser over other browsers in order to protect your privacy while visiting our websites.
```
